<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menú Saludable Inteligente</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="iniciarSesion">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar sesión</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bienvenido usuario, digite su nombre de usuario y contraseña:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreUsuario">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordUsuario">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="crearUsuario">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear usuario</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
